docs/source/known-issues.rst

```diff
@@ -1,6 +1,6 @@
-============================
-Known issues and limitations
-============================
```

Comment:
To me "Known issues" are problems that could be fixed, while "limitations" can't be fixed, so I'd prefer to keep both terms.

Reply:
I only shortened for the nav so no objection to keeping both terms